Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml

```diff
@@ -25,6 +25,14 @@
   - type: Fixtures
     fixtures:
       fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.35,-0.35,0.35,0.35"
+        mask:
+        - Impassable
+        - HighImpassable
+        - MidImpassable
+      fix2:
         shape:
           !type:PhysShapeAabb
           bounds: "-0.35,-0.35,0.35,0.35"
```

Comment:
Can't you just make the fixture hard below?

Reply:
No, it needs two fixtures; one hard fixture for generating contacts and one non-hard fixture to detect artifacts that have been placed on the machine.